spectral_cube/tests/test_spectral_cube.py

```diff
@@ -344,6 +342,40 @@ def test_apply_everywhere(self, operation, value, data_advs, use_dask):
         # allclose fails on identical data?
         #assert_allclose(d1o, c1o.filled_data[:])
 
+
+    @pytest.mark.parametrize(('operation', 'value'), ((operator.add, 0.5*u.K),
+                                                      (operator.sub, 0.5*u.K),))
+    def test_apply_everywhere_plusminus(self, operation, value, data_advs, use_dask):
+        c1, d1 = cube_and_raw(data_advs, use_dask=use_dask)
+
+        assert c1.unit == value.unit
+
+        # append 'o' to indicate that it has been operated on
+        # value.value: the __add__ function explicitly drops the units
+        c1o = c1._apply_everywhere(operation, value.value, check_units=False)
```

Comment:
Add a test case with `check_units=True`?

Reply:
`add` is only supposed to use `check_units=False`; multiply/divide use `check_units=True`